packages/node_modules/@webex/plugin-meetings/sample/app.js

```diff
@@ -164,12 +164,26 @@ function disconnect() {
   return Promise.resolve();
 }
 
-function getParticipantString(list) {
-  let string = '';
-  list.forEach((p) => {
-    string += `${p.person.email},`;
-  });
-  return string;
+function getParticipantNameString(participant) {
```

Comment:
This should change to "member" right?

Reply:
this is in app.js, we can call it whatever lol but yeah member is fine